docs/contributors/release-process.md

```diff
@@ -39,3 +48,13 @@ git rebase -i origin/master
 git tag -as v0.1 -m "Initial release."
 git push origin v0.1
 ```
+
+### After the release
+
+1. Announce the release on Twitter, Discord and other channels.
+2. Deploy the release:
+   - Inform clients
+   - Deploy release to the `wakuv2.prod` fleet from [Jenkins](https://ci.status.im/job/nim-waku/job/deploy-v2-prod/).
```

Comment:
I wonder if we want to use some kind of green/blue setup here in future? So we keep a few nodes running with previous release and make the thing less coordinated (more realistic in p2p environment).

Reply:
I agree with that suggestion. Doing a full fleet upgrade can be a disruptive action for the service. Doing it in a gradual way is what makes sense for me.

We can discuss if we want to follow a blue-green or canary rollout strategy (see: [Canary release strategy vs. Blue/Green](https://stackoverflow.com/a/72161773/1099999)).  

But let's keep in mind that doing a gradual rollout strategy also will require:
* Defining the different rollout stages (e.g. stage1: v1-90% v2-10, stage2: v1-50% v2-50%, and so on).
* Defining the criteria for considering a rollout successful and defining rollback criteria and strategy in the case the partial rollout does not go well. This may have some functional requirements for the nodes to provide the right metrics.
* Defining which parts of the rollout process should be automatic.
* etc.